test/unit/uk/gov/hmrc/bindingtariffrulingfrontend/config/AppConfigTest.scala

```diff
@@ -54,7 +54,8 @@ class AppConfigTest extends BaseSpec {
   }
 
   "Build beta feedback unauthenticated url" in {
-    appConfig("contact-frontend.host" -> "host").betaFeedbackUnauthenticatedUrl shouldBe "host/contact/beta-feedback-unauthenticated?service=AdvanceTariffRulings"
+    appConfig("contact-frontend.host" -> "host").betaFeedbackUnauthenticatedUrl shouldBe
+      "host/contact/beta-feedback-unauthenticated?service=AdvanceTariffRulings"
```

Comment:
Extract "host/contact/beta-feedback-unauthenticated?service=AdvanceTariffRulings" into a variable and name it expectedFeedbackURL and use that after shouldBe, code will be more readable